src/js/models/list-item.js

```diff
@@ -0,0 +1,51 @@
+import Section from './markup-section';
+import LinkedList from '../utils/linked-list';
+
+export const LIST_ITEM_TYPE = 'list-item';
+
+export default class ListItem extends Section {
+  constructor(tagName, markers=[]) {
+    super(tagName);
```

Comment:
Yeah, that would work. Originally the constructors for these two were not so similar, but later refactorings removed the differences. I'll change this.